bin/collator/src/parachain/chain_spec/shiden.rs

```diff
@@ -160,6 +150,39 @@ fn make_genesis(
         assets: Default::default(),
         parachain_system: Default::default(),
         transaction_payment: Default::default(),
+        dapp_staking: DappStakingConfig {
+            reward_portion: vec![
```

Comment:
reward_portion, slot_distribution and  tier_thresholds seems like shibuya values. Please update